typing/types.mli

```diff
@@ -523,6 +523,24 @@ val create_row:
   name:(Path.t * type_expr list) option -> row_desc
 
 val row_fields: row_desc -> (label * row_field) list
+
+(** [row_more] returns a [type_expr] with one of the following [type_desc]s:
+
+    * [Tvar]: This is a standard row variable; it would occur in e.g. [val f :
+    [> `A | `B] -> int]. When/if we learn more about a polymorphic variant, this
+    variable might get unified with one of the other [type_desc]s listed here,
+    or a [Tvariant] that represents a new set of constructors to add to the row.
+
+    * [Tunivar]: This is a universally quantified row variable; it would occur
+    in e.g. [type t = { f : 'a. ([> `A | `B ] as 'a) -> int }].
+
+    * [Tconstr]: This is always an abstract [#row] type created by a [private]
+    row type, as in [type t = private [> `A | `B]]. (After a substitution, this
+    type might have a manifest; expanding the type will yield either a
+    [Tvariant] or another [type_desc] listed here.
```

Comment:
This could also be an abstract row coming from an existential

Reply:
Yes, Jacques pointed this out, too. See https://github.com/ocaml/ocaml/pull/14023